src/dal/views.py

```diff
@@ -55,6 +55,7 @@ def has_more(self, context):
 
     def get_result_value(self, result):
         """Return the value of a result."""
+        # return six.text_type(result.pk)
```

Comment:
[Test 199524496](https://travis-ci.org/yourlabs/django-autocomplete-light/builds/199524496) is failing on Django 1.10 *without* my change.

For future reference, the diff for this test run is this:

```diff
commit 1dd93245bf8e44fdd9a3a05ca7a30082f0e2794c
Author: blag
Date:   Mon Feb 6 12:23:04 2017 -0600

    Support serializing UUIDs - closes #781

diff --git a/src/dal/views.py b/src/dal/views.py
index b23aa5c..57824dd 100644
--- a/src/dal/views.py
+++ b/src/dal/views.py
@@ -55,6 +55,7 @@ class BaseQuerySetView(ViewMixin, BaseListView):

     def get_result_value(self, result):
         """Return the value of a result."""
+        # return six.text_type(result.pk)
         return result.pk

     def get_result_label(self, result):
```